docroot/profiles/custom/cgov_site/modules/custom/cgov_metatag/cgov_metatag.tokens.inc

```diff
@@ -77,12 +81,33 @@ function cgov_metatag_tokens($type, $tokens, array $data, array $options, $bubbl
         case 'cgov-media-content-language':
           $replacements[$original] = get_media_meta_content_language_token($data);
           break;
+
+        case 'cgdp-field-pdq-is-svpc':
```

Comment:
This should be moved into the PDQ module where this field is defined. (should be pdq_cancer_information_summary)

Reply:
Created new .inc file in pdq_cancer_information_summary.